CIScripts/Build/Zuul.ps1

```diff
@@ -35,10 +36,11 @@ function Clone-ZuulRepos {
             zuul-cloner.exe @ZuulClonerOptions @ProjectList
         }
         
-        # TODO: Use Juniper repo: git clone contrail-windows-docker-driver.github.com:Juniper/contrail-windows-docker-driver.git
+        $DriverClonePath = $DriverSrcPath -replace "github.com/", "github.com:"
```

Comment:
This feature seems to be unrelated to cloning from zuul/gerrit. To avoid reader's confusion it should be done directly in Build.ps1. Clone-Repos function might be a useful tool.